code/style.css

```diff
@@ -1,7 +1,574 @@
+  line-height: 01em;
+  text-align: center;
+}
+
+.hero-image-text h3 {
+  font-family: 'Raleway', sans-serif;
+  color:#0e2708;
+  font-size: 20px;
+  font-weight: 200;
+}
+
+header button {
+  height: 50px;
+  border: none;
+  background: #f0c697;
+  color: #ffffff;
+  padding: 10px;
+  margin: auto;
+  position: relative;
+  box-sizing: border-box;
+}
+
+header button:hover {
+  cursor: pointer;
+  height: 60px;
+  width: 90px;
+  transition: 2s;
+}
+
+
+/* MAIN CONTAINER WITH FORM & ACCORDIAN */
+main {
+  width: 100%; 
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  margin-top: 550px;
+}
+
+/* FORM */
+.form-parent {
+  width: 90%;
+  padding: 20px;
+  margin: 20px auto;
+  box-sizing: border-box;
+  background: #dadfd9;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.input-section {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.input-section h3 {
+  padding-bottom: 5px;
+  margin-bottom: 20px;
+  margin-top: 0px;
+  border-bottom: solid #31482c 1px;
+  font-weight: 100;
+  font-family: 'Rozha One', serif;
+  text-align: center;
+  font-size: 22px;
+  color: #31482c;
+}
+
+.input-section label {
+  margin-bottom: 15px;
+}
+
+.input-section input {
+  width: 250px;
+  height: 40px;
+  margin-bottom: 10px;
+  border: none;
+  background: #ffffff;
+}
+
+.input-section input::placeholder {
+  padding-left: 10px;
+  color: #90be6b;
+  font-weight: 200;
+}
+
+/* RADIO BUTTON CONTAINER */
+
+.radio-button-column {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
+
+.radio-button-column h3 {
+  text-align: center;
+  margin: 5px 0px 15px 0px;
+  padding-bottom: 5px;
+  border-bottom: solid #31482c 1px;
+  font-weight: 100;
+  font-family: 'Rozha One', serif;
+  font-size: 22px;
+  color: #31482c;
+}
+
+.radio-button-container label {
+  font-family: 'Raleway', sans-serif;
+  color: #31482c;
+}
+  
+.radio-button-row {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 250px;
+}
+
+.radio-button-container {
+  display: block;
+  position: relative;
+  padding-left: 25px;
+  margin: 10px 5px;
+  font-size: 16px;
+  user-select: none;
+}
+
+.radio-button-row label {
+  color: #31482c;
+}
+
+.radio-button-container input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+.custom-radio {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 19px;
+  width: 19px;
+  background-color:#ffffff;
+  border-radius: 50%;
+  cursor: pointer;
+}
+
+.radio-button-container:hover input ~ .custom-radio {
+  background-color: rgb(230, 243, 229);
+}
+
+.radio-button-container input:checked ~ .custom-radio {
+  background-color: #f0c697;
+}
+
+.custom-radio:after {
+  content:"";
+  position: absolute;
+  top: 6px;
+  left: 6px; 
+  width: 7px;
+  height: 7px;
+  background-color: #ffffff;
+  border-radius: 50%;
+  display: none;
+}
```

Comment:
Notice how the `custom-radio` naming looks a bit out of place here compared to the `radio-` above and below?